bin/cs-bdd

```diff
@@ -127,21 +129,138 @@ end
 
 module CS
   module BDD
+    class Runner < Thor
+      desc 'ios [IP_ADDRESS] [PROFILE]',
+           'Run tests in IOS.'
```

Comment:
Aqui eu colocaria `specs` ao invés de `tests`, faz mais sentido `Run all specs for iOS`.